python/ray/_private/runtime_env/agent/main.py

```diff
@@ -0,0 +1,197 @@
+import sys
+import os
+import argparse
+import logging
+import pathlib
+import ray._private.ray_constants as ray_constants
+from ray.core.generated import (
+    runtime_env_agent_pb2,
+)
+from ray._private.utils import open_log
+from ray._private.ray_logging import (
+    configure_log_file,
+)
+from ray._private.utils import (
+    get_or_create_event_loop,
+)
+
+
+def import_libs():
+    my_dir = os.path.abspath(os.path.dirname(__file__))
+    sys.path.insert(0, os.path.join(my_dir, "thirdparty_files"))  # for aiohttp
+    sys.path.insert(0, my_dir)  # for runtime_env_agent and runtime_env_consts
+
+
+import_libs()
+
+import runtime_env_consts  # noqa: E402
+from runtime_env_agent import RuntimeEnvAgent  # noqa: E402
+from aiohttp import web  # noqa: E402
```

Comment:
Q: how does it work with aiohttp dependency downloaded by `ray[default]`?

Reply:
we prepend the sys path so it will always use vendored `aiohttp` regardless of the customer's environment